variables.tf

```diff
@@ -47,19 +47,18 @@ variable "private_subnets" {
   type = list(object({
     name                 = string,
     nat                  = bool,
-    vsphere_service_type = string,
+    vsphere_service_type = optional(string),
     routable             = bool,
     cidr                 = string,
-    reserved_ip_count    = optional(number)
+    reserved_ip_count    = optional(number, 100)
```

Comment:
I verified that this `reserved_ip_count` default of 100 would only applied when `routable` is set.
The net effect of this is that the DHCP assignable range will default to 100 less than the max IP in the `cidr`. 

`{ "routable": true, "cidr": "10.0.0.0/24" }` -> `reserved_ip_count: 100` -> DHCP Max Assigned address: `10.0.0.155`

Without this `100`, the default defined at https://github.com/equinix/terraform-metal-vsphere/blob/cdea3e3a750d89f0d0526dbb0f340563447be6ff/templates/pre_reqs.py#L118 would be a `reserved_ip_count` that consumes the remainder of the assignable space in the CIDR.

With this change to `100`, that line of python becomes dead code.  Why would 100 be a better default than the existing dynamic default?

Reply:
This doesn't change the existing default, just the notation. 
From `"reserved_ip_count" : 100`
to `reserved_ip_count    = optional(number, 100)`